SDWebImage/Core/UIImage+Transform.m

```diff
@@ -566,7 +566,7 @@ - (nullable UIImage *)sd_tintedImageWithColor:(nonnull UIColor *)tintColor {
     CGFloat scale = self.scale;
     
     // blend mode, see https://en.wikipedia.org/wiki/Alpha_compositing
-    CGBlendMode blendMode = kCGBlendModeSourceAtop;
+    CGBlendMode blendMode = kCGBlendModeSourceIn;
```

Comment:
Need new `CGBlendMode` and fix for `CIFilter` code path as well.

CI filter: https://developer.apple.com/library/archive/documentation/GraphicsImaging/Reference/CoreImageFilterReference/index.html

Reply:
SDWebImage/Core/UIImage+Transform.m